public/main/inc/lib/api.lib.php

```diff
@@ -7408,73 +7408,38 @@ function api_create_zip(string $name): ZipStream
  */
 function api_get_language_translate_html()
 {
-    $translate = api_get_configuration_value('translate_html');
+    $translate = (api_get_setting('editor.translate_html') === 'true');
```

Comment:
Use "yoda" style: 

`if ('true' === api_get_setting('editor.translate_html'))`

Reply:
Check the results 

https://github.com/chamilo/chamilo-lms/runs/3060987193?check_suite_focus=true